src/components/Modals/NewCourse/NewCourseModal.vue

```diff
@@ -60,31 +95,74 @@ export default Vue.extend({
     isCourseModelSelectingSemester: Boolean,
     reqs: Array as PropType<readonly SingleMenuRequirement[]>,
```

Comment:
Can we do `type` and `required` for the props declarations?

Reply:
changed